common/protob/messages-cardano.proto

```diff
@@ -210,6 +215,9 @@ message CardanoSignTxInit {
     optional bytes script_data_hash = 16;
     required uint32 collateral_inputs_count = 17;
     required uint32 required_signers_count = 18;
+    required bool has_collateral_return = 19;
```

Comment:
Yeah well, we introduced the collateral input type so that it's easier to add new attributes in case the chain adds some (but it seems rather unlikely to me atm). I added the reference input type to be consistent with collateral inputs, but adding a new type for collateral return when it's just a more constrained output (and is likely to stay that way) seems too far-fetched to me :thinking: What do you think?

Reply:
I rather agree it's too far-fetched. Maybe we should have reused the inputs message previously but it doesn't matter now. I was just curious what you thought.